### PR TITLE
Make wait time seconds modifiable

### PR DESCRIPTION
**Pull request by seanliu-oss**
Currently SQSMessageConsumerPrefetch.WAIT_TIME_SECONDS is a hardcoded constant of 20 seconds, which makes shutdown time lengthy for tests such as SpringBoot Integration tests. Also because it is static final it cannot be changed via Reflection.

This PR changes it to protected static, so if required the value can be updated via reflection for testing purposes.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ziyanli-amazon** [comment]
Thanks. I will take a look at it to see if this has been used elsewhere, but LGTM overall.

File: src/test/java/com/amazon/sqs/javamessaging/ModifyWitTimeSecondsTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.amazon.sqs.javamessaging;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ModifyWaitTimeSecondsTest {
```
Comment (ziyanli-amazon) [inline src/test/java/com/amazon/sqs/javamessaging/ModifyWitTimeSecondsTest.java:10]: Can you fix the typo in the java file name please?

Comment (seanliu-oss) [reply src/test/java/com/amazon/sqs/javamessaging/ModifyWitTimeSecondsTest.java]: Fixed.

**Comment by pzygielo** [comment]
- #114

**Pull request merged**